src/org/zaproxy/zap/authentication/FormBasedAuthenticationMethodType.java

```diff
@@ -273,6 +274,7 @@ public WebSession authenticate(SessionManagementMethod sessionManagementMethod,
 		 * Sets the login request as being an existing SiteNode.
 		 * 
 		 * @param loginSiteNode the new login request
+		 * @throws Exception if an occurred while obtaining the message from the node
```

Comment:
It's missing "error", will add that.

Yes, `getHttpMessage()` throws some exceptions.